examples/console_log_example.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022 Google LLC.
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script implements Puppeteer's `examples/cross-browser.js` scenario using WebDriver BiDi.
+# https://github.com/puppeteer/puppeteer/blob/4c3caaa3f99f0c31333a749ec50f56180507a374/examples/cross-browser.js
+
+import asyncio
+import logging
+from pathlib import Path
+from _helpers import *
+
+logging.basicConfig(
+    format="%(message)s",
+    level=logging.DEBUG,
+)
+
+
+async def main():
+    ## Part 1. Open a page.
+
+    # Open browser
+    websocket = await get_websocket()
+    await run_and_wait_command({
+        "id": 0,
+        "method": "session.new",
+        "params": {}}, websocket)
+
+    # Open tab
+    command_result = await run_and_wait_command({
+        "id": 1000,
+        "method": "browsingContext.create",
+        "params": {
+            "type": "tab"
+        }}, websocket)
+    # `command_result` should be like this:
+    # {
+    #     "id": 1000,
+    #     "result": {
+    #         "context": "__SOME_CONTEXT_ID__",
+    #         "url": "",
+    #         "children": []
+    #     }
+    # }
+    context_id = command_result['result']['context']
+
+    # Navigate to page
```

Comment:
This step can be omitted, as we evaluate custom script and don't need any specific page. From the other side, users will likely need to load some page before they make their tests. WDYT?

Reply:
How about `about:blank`? Alternatively, `data:text/html,<h1>Hello world</h1>`.

Reply:
`data:text/html,<h1>Hello world</h1>` LGTM